Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml

```diff
@@ -468,15 +485,32 @@ script:
         if (params.eventsQuery){
             query = params.eventsQuery;
         }
-        var fields = null;
+        if (params.fields){
+            var fields = params.fields
+            var discover_fields = true;
+        } else {
+            var fields = null;
+            var discover_fields = false
```

Comment:
Let's add a semicolon at the end of each line

Reply:
done